CrossClusterRedo393 (#349)
tigergraph/server-docs: modules/backup-and-restore/pages/cross-cluster-backup.adoc

* You have access to the TigerGraph Linux user account on your cluster.
All commands must be run from the TigerGraph Linux user.
* You have access to a data backup from another cluster.
* [xref:_procedure_v3_9_1_and_below[3.9.1 and earlier]] You have the metadata backup that goes with the data backup.
* You have xref:configurations.adoc[configured backup and restore] on the new cluster.

== Procedure

=== 1) Copy Data Backup to a New Cluster

==== For Amazon S3 Backup:

> If your backups are stored in Amazon S3 buckets, you can skip this step as long as you configure:

Reviewer: ```suggestion
If your backups are stored in Amazon S3 buckets, set the following configuration parameter to be the location of your backup files.
```
I think your statement is technically correct, but it's phrased in a backward, exception-case way.
"You don't need to do anything, unless you do."  --> Just be direct and tell me what to do.

Author: Got it! Thank you. I will keep this in mind. Updated with latest commits.

[source,console]
System.Backup.S3.BucketName

For the new cluster to be the same as the previous cluster

==== For Local Backup Storage:

If your backups were stored locally, you need to make sure backup files from all nodes are copied to the new cluster.
They do not necessarily need to be on the corresponding nodes as long as all partitions are in the new cluster at the specified paths you configured.

* For example:
+

> .if you have configured the backup path to be:

Reviewer: ```suggestion
.If you have configured the backup path to be
```
No colon because the code is actually part of the sentence. (If you wrote it all on one line, you wouldn't insert a colon.)

Author: updated with latest commits. Thank you.

[source,console]
/home/tigergraph/backups
+

> .For a 3-node cluster, you can put all backup files for backup:

Reviewer: ```suggestion
.for a 3-node cluster, you can put all backup files for backup
```
No capitalization and no colon, because this is a fragment in the middle of a sentence.  Yes, this is actually a sentence, which should follow regular grammar rules. The sentence just happens to have code examples as some of its nouns:
_If you have configured the backup path to be `<this>` for a 3-node cluster, you can put all the backup files for backup `<name of your backup folder>` in the folder like this:
`<path for where to put the backup folder>`_

Author: updated with latest commits. And good tip, I will keep it in mind. Thank you.

[source,console]
weekly-<timestamp>
+

> .In the folder like this:

Reviewer: ```suggestion
.in the folder like this:
```
No capitalization, but a colon because we're reached the end of the sentence.

Author: updated with latest commits. Thank you.
